inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -739,11 +742,12 @@ linelist_cleaned$age_group <- age_categories(linelist_cleaned$age_years,
 
 ## to combine different age categories use the following function 
 ## this prioritises the smaller unit, i.e. if given months and years, will return months first
-## generally, these delineations are NOT used for AJS.
+## generally, these delineations are NOT used for measles.
```

Comment:
ajs was correct here

Reply:
This is super weird. I don't remember making this change!